airdrop/alembic/versions/22cd02190a1c_addstakeLimitThreshold.py

```diff
@@ -0,0 +1,52 @@
+"""AddBlockNumberOnRegistrationTable
+
+Revision ID: 22cd02190a1c
+Revises: 7ebcbe8dcfa1
+Create Date: 2022-07-26 10:57:29.400285
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = '22cd02190a1c'
+down_revision = '7ebcbe8dcfa1'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Why do we have alter commands for other columns such as claim_end_period, claim_start_period

Reply:
fixed now